pyqtgraph/graphicsItems/PColorMeshItem.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import division
+
+from ..Qt import QtGui, QtCore
+import numpy as np
+from .. import functions as fn
+from .. import debug as debug
+from .GraphicsObject import GraphicsObject
+from ..Point import Point
+from .. import getConfigOption
+from .GradientEditorItem import Gradients # List of colormaps
+from ..colormap import ColorMap
+
+try:
+    from collections.abc import Callable
+except ImportError:
+    # fallback for python < 3.3
+    from collections import Callable
+
+__all__ = ['PColorMeshItem']
+
+
+class PColorMeshItem(GraphicsObject):
+    """
+    **Bases:** :class:`GraphicsObject <pyqtgraph.GraphicsObject>`
+    """
+
+    sigImageChanged = QtCore.Signal()
+    sigRemoveRequested = QtCore.Signal(object)  # self; emitted when 'remove' is selected from context menu
+
```

Comment:
Nitpick: there are extra lines between methods and such in this class that could be removed.